eng/tox/tox.ini

```diff
@@ -167,11 +167,12 @@ commands =
       -p {tox_root} \
       -w {envtmpdir} \
       --package-type sdist
+    python {repository_root}/eng/tox/create_dependencies_and_install.py -p {tox_root}
```

Comment:
same as https://github.com/Azure/azure-sdk-for-python/pull/28641 but for mypy. Matches local to CI run (source of truth)